tidy3d/web/api/webapi.py

```diff
@@ -640,15 +638,7 @@ def delete_old(
     folder = Folder.get(folder)
     if not folder:
         return 0
-    tasks = folder.list_tasks()
-    if not tasks:
-        return 0
-    tasks = list(
-        filter(lambda t: t.created_at < datetime.now(pytz.utc) - timedelta(days=days_old), tasks)
-    )
-    for task in tasks:
-        task.delete()
-    return len(tasks)
+    return folder.delete_old(days_old)
```

Comment:
if the data is large, for example, 10K tasks.  will the webapi get time out with new code?
I would like to see the test result to confirm it.

Reply:
another question is how we tell user the progress during the deletion.
use 30K task as examples since it is current max task count right now.